content/blog/2015-05-24-custom-sd.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Plugging in your Custom Service Discovery
+created_at: 2015-05-24
+kind: article
+author: Fabian Reinartz
+---
+
+This week we released Prometheus v0.14 — a version with many long-awaited additions
+and improvements.
+
+On the user side, the the new means for service discovery aim to make it easier to
```

Comment:
"means" -> "mechanisms" is a little easier to read because "mechanisms" can't be a verb too. Also sounds better in that context in general IMO.